<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectParentLayoutDialog</class>
 <widget class="QDialog" name="SelectParentLayoutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Layout</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Version Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="le_version_name"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="l_select_layout_prompt">
       <property name="text">
        <string>Parent Layout:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cmb_select_layout">
       <property name="insertPolicy">
        <enum>QComboBox::InsertAlphabetically</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="l_file_name_prompt">
       <property name="text">
        <string>File Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="le_file_name"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_file_name</tabstop>
  <tabstop>le_version_name</tabstop>
  <tabstop>cmb_select_layout</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectParentLayoutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectParentLayoutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmb_select_layout</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>SelectParentLayoutDialog</receiver>
   <slot>selection_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selection_changed(int)</slot>
 </slots>
</ui>
